weave/trace/weave_client.py

```diff
@@ -776,13 +782,6 @@ def send_end_call() -> None:
 
         self.future_executor.defer(send_end_call)
 
-        # Descendent error tracking disabled til we fix UI
-        # Add this call's summary after logging the call, so that only
-        # descendents are included in what we log
-        # summary.setdefault("descendants", {}).setdefault(
-        #     call.op_name, {"successes": 0, "errors": 0}
-        # )["successes"] += 1
```

Comment:
is this fixed?

Reply:
No, this code has been commented out since it entered the code base six months ago in https://github.com/wandb/weave/pull/1414 

I could leave it in if you'd prefer.

Reply:
@tssweeney do we need this?

Reply:
There's a screenshot in the PR I linked above that illustrates why this never made it in.

There's a redesign for the Summary tab in the queue, so if we want to revive this it's probably a good time.
https://www.notion.so/wandbai/Trace-drawer-items-43c3106f7d2848758d02f89cee2b3540?pvs=4#10ae2f5c7ef3808e8e5de7424dfa0584

We have a long standing issue where we try to calculate the existence of descendent errors in the UI to show a different status pill, but we don't do it in a consistent way. If we were going to log this kind of stat in the summary we could use it for that. I think it should be scoped under the "weave" key if we do that.

Reply:
Ok I would keep this comment and also include reference to your doc future readers' clarity

Reply:
I can do that.